adminservice/src/main/java/nu/yona/server/admin/batch/config/ServletConfiguration.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nu.yona.server.admin.batch.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ImportResource;
+
+/**
+ * Just the import of the servlet-config.xml of spring-batch-admin-manager.
+ *
+ * @author Thomas Bosch
+ */
+@Configuration
+@ImportResource("classpath:/org/springframework/batch/admin/web/resources/servlet-config.xml")
```

Comment:
Why not configure it with methods instead? That would be more consistent with all other configuration classes.